cmd/buildctl/build.go

```diff
@@ -35,8 +35,9 @@ var buildCommand = cli.Command{
 			Usage: "Define custom options for exporter",
 		},
 		cli.BoolFlag{
-			Name:  "no-progress",
-			Usage: "Don't show interactive progress",
+			Name:   "console",
+			Usage:  "Show console output",
+			Hidden: true,
```

Comment:
Why is it hidden?

Also, shouldn't this default to true so the autodetection of a tty on stdio and, if possible, the fancy output, happens by default? (Maybe `s/Hidden/Default/g` is what you meant?).

Reply:
Yes, Hidden was meant for Default. Since `BoolFlag` had no field `Value` or `Default` so I used `Hidden`.

I will update PR with autodetection of a tty instead of making true by default.